src/js/editInLocalStorage.js

```diff
@@ -0,0 +1,24 @@
+import createElement from './createElement.js';
+import List from './list.js';
+
+const editDescription = (index, icon) => {
+  const input = document.querySelector('.edit-todo');
+  const list = new List();
+  input.addEventListener('keydown', (e) => {
+    if (e.key === 'Enter') {
+      list.UpdateDescriptionTask(Number(index), input.value);
+      const task = list.getTaskByIndex(Number(index));
+      const parent = input.parentNode;
+      input.remove();
+      const label = createElement('label');
+      const inputCheck = createElement('input', task);
+      const span = createElement('span', task);
+      label.appendChild(inputCheck);
+      label.appendChild(span);
+      parent.insertBefore(label, parent.firstChild);
+      icon.classList.add('fa-ellipsis-vertical');
+      icon.classList.remove('fa-trash-can');
+    }
+  });
+};
+export default editDescription;
```

Comment:
- [ ] Well done :clap: I really appreciated your effort :+1: but there is some problem with editing functionality when someone is editing any of the todos it shows `undefined` kindly fix this issue :+1: check below :point_down: the video for more clarification :slightly_smiling_face: 

[screen-capture (2).webm](https://user-images.githubusercontent.com/45226848/236616520-20c5a868-128e-4070-b523-196e64eefd5e.webm)